scripts/clear-certain-fault-leds.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# This shell script sets Functional property to true for
```

Comment:
Having "certain FRUs" can lead to ambiguity in future. As discussed we will keep it very specific to PSU fault, kindly update the comments and it will be good if we can have very specific name for the script as well.  "clear-PSU-fault-leds.sh"
IMO that will have better clarity.

Reply:
done